Library/Homebrew/os/mac.rb

```diff
@@ -41,6 +41,10 @@ def cat
       version.to_sym
     end
 
+    def language
+      @language ||= Utils.popen_read('defaults', 'read', '.GlobalPreferences', 'AppleLanguages').delete(" \n\"()").sub(/,.*/, '')
```

Comment:
Speaking of RuboCop, when was the last time someone actually ran it? Seems there are thousands of violations.